prod/westeurope/internal/api/apim/api_management_api_cgn_merchant/swagger.json.tmpl

```diff
@@ -0,0 +1,122 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.1.0",
+    "title": "IO Functions CGN Merchant",
+    "contact": {
+      "name": "IO team",
+      "url": "https://forum.italia.it/c/progetto-io"
+    },
+    "x-logo": {
+      "url": "https://io.italia.it/assets/img/io-logo-blue.svg"
+    },
+    "description": "This Specs are intended for all CGN operators that must integrate their e-commerce portal through CGN API discounts system.\n"
+  },
+  "host": "${host}",
+  "basePath": "/api/v1",
+  "schemes": [
+    "https"
+  ],
+  "security": [
+    {
+      "ApiKey": []
+    }
+  ],
+  "paths": {
+    "/merchant/cgn/otp/validate": {
+      "post": {
+        "operationId": "validateOtp",
+        "summary": "Validate a CGN OTP\n",
+        "description": "Validate an OTP related to a CGN (Carta Giovani Nazionale).\nThis endpoint could be called for 2 different use cases:\n- When an operator wants just to verify an OTP\n- When an operator wants to verify an OTP and invalidate it at the same time (i.e: checkout completed)\n",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/ValidateOtpPayload"
+            },
+            "description": "The request wrapper of a user data processing choice."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OTP validation success.",
+            "schema": {
+              "$ref": "#/definitions/OtpValidationResponse"
+            }
+          },
+          "401": {
+            "description": "Wrong or missing API key."
+          },
+          "403": {
+            "description": "Forbidden."
+          },
+          "404": {
+            "description": "OTP Not Found/Invalid"
+          },
+          "500": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/ProblemJson"
+            }
+          }
+        }
+      }
+    }
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "ApiKey": {
+      "type": "apiKey",
+      "name": "X-Functions-Key",
```

Comment:
@AleDore I am not sure the security definition here in the swagger file is correct.
If you double check with the bonus vacanze of the function service the expected key is **Ocp-Apim-Subscription-Key**

Reply:
Fixed on [master](https://github.com/pagopa/io-functions-cgn-merchant/blob/master/openapi/index.yaml) branch.

Reply:
updated